# Generated for NaoQi method ALAudioPlayer.stopAll
# Stops all the files that are currently playing.

---


